exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -71,8 +70,8 @@ func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem
 	// Correctly sets apiSubmitLogRequest ddtags field based on tags from translator Transform method
 	if payload[0].HasDdtags() {
 		tags := datadog.PtrString(payload[0].GetDdtags())
-		if s.opts.Ddtags != nil {
-			tags = datadog.PtrString(fmt.Sprint(*s.opts.Ddtags, ",", payload[0].GetDdtags()))
+		if s.verbose {
+			s.logger.Debug("Tags", zap.String("tags", *tags))
 		}
 		s.opts.Ddtags = tags
```

Comment:
The bug is here. We need to clone the opts and append tags . Can you please modify the same

Reply:
We need to append Ddtags that are given in the opts as well.